manual/core/integration/README.md

```diff
@@ -0,0 +1,287 @@
+## Integration
```

Comment:
Should add a link to beginning of manual/core/README.md that links here as that currently offers how to add dependency to maven, but would also be good to point here for more information.